src/platform/nrfconnect/wifi/WiFiManager.cpp

```diff
@@ -144,6 +144,33 @@ const Map<uint32_t, WiFiManager::NetEventHandler, 5> WiFiManager::sEventHandlerM
     { NET_EVENT_WIFI_DISCONNECT_COMPLETE, WiFiManager::DisconnectHandler },
 });
 
+void WiFiManager::WiFiSupplicantEventHandler(net_mgmt_event_callback * cb, uint32_t mgmtEvent, net_if * iface)
+{
+    switch (mgmtEvent)
+    {
+    case NET_EVENT_WPA_SUPP_READY:
```

Comment:
Is this possible that any of these events is generated after we successfully initialized supplicant/iface? I wonder if it would make any sense to process them and inform an application that something went wrong.

Reply:
I thought about that, but it seems that we need to wait ~50-100 ms to receive those events after initializing our WiFi Manager.